pkg/apis/flagger/v1beta1/canary.go

```diff
@@ -403,6 +403,11 @@ type CanaryWebhookPayload struct {
 	// Phase of the canary analysis
 	Phase CanaryPhase `json:"phase"`
 
+	// Hash from the TrackedConfigs and LastAppliedSpec of the Canary.
+	// Can be used to identify a Canary for a specific configuration of the
+	// deployed resources.
+	Checksum string `json:"checksum"`
```

Comment:
Please add this field to the payload example in https://github.com/fluxcd/flagger/blob/main/docs/gitbook/usage/webhooks.md and explain what it does, you can use the same text as in the comment.

Reply:
Done, but this does raise the question, should I add this to the EventWebhook too?

I don't think I need it for what I'm doing, but...it feels logical to add there too?